examples/camera/basic/buildozer.spec

```diff
@@ -31,8 +31,9 @@ source.include_exts = py,png,jpg,kv,atlas
 # (str) Application versioning (method 2)
 version = 1.0
 
-# (list) Application requirements
-requirements = plyer,kivy
+# (list) Application requirements 
+# android is also require for run this app on android
+requirements = plyer,kivy,android
```

Comment:
I don't think there is need to mention android in `requirements` since this buildozer.spec can also be used to create iOS app. Just informing the user with the help of comment will be enough. 
`requirements` should contain the common libraries which are required for both Android and iOS.

Reply:
ok, I have done it.
now it should be fine.